metadata-ingestion/examples/demo_data/demo_data.json

```diff
@@ -84,7 +87,8 @@
                                     },
                                     "nativeDataType": "Integer()",
                                     "recursive": false,
-                                    "globalTags": null
+                                    "globalTags": null,
+                                "globalGlossaryTerms": null
```

Comment:
nit: looks like the indentation here is a bit off

Reply:
done